src/fromevent.js

```diff
@@ -39,13 +42,21 @@ var findHandlerMethods = function(target) {
   throw new Error("No suitable event methods in " + target);
 };
 
-Bacon.fromEventTarget = function(target, eventName, eventTransformer) {
+Bacon.fromEventTarget = function(target, eventName, ...args) {
+  let optionsOrUseCapture, eventTransformer;
+
+  if (_.isFunction(args[0])) {
+    [eventTransformer] = args;
+  } else {
+    [optionsOrUseCapture, eventTransformer] = args;
+  }
```

Comment:
Yes, `addEventListener` added `options|useCapture` to the end of the parameter list. If we did the same in Bacon, it would be `Bacon.fromEvent(target, listener, [eventTransformer [, options|useCapture]])`.

For the following reasons, I didn't add `options|useCapture` to the end:

* I think Bacon wants `eventTransformer` always at the end.
* I'm not a fan of `eventTransformer` (a Bacon concern) coming between `target`, `listener` and `options|useCapture` (all `addEventListener` concerns) seems awkward.
* It would be a breaking change.

I'm not comfortable with `Bacon.fromEvent(element, 'click', {eventTransformer: () => {}, passive: true});` because we'll have to `omit` `eventTransformer` from `options` before passive it to `addEventListener`. `addEventListener` will ignore it, but it's not really clean for that property to bleed into `addEventListener`.

Reply:
> argument order and checking will only be maintainable for so long

e.g. if `eventTransformer` in a future version of Bacon.js can be something other than a function?

Reply:
Yeah, I don't like the idea of splitting out non dom options from an object.